packages/react-router-dom/__tests__/data-browser-router-test.tsx

```diff
@@ -4434,72 +4475,6 @@ function testDomRouter(
         `);
       });
 
-      it("renders hydration errors on lazy leaf elements", async () => {
```

Comment:
This is not a use-case we need to support (for now) since we landed on SSR needing to pre-resolve `lazy` for initial matches